changelog/8.9.0.yaml

```diff
@@ -0,0 +1,86 @@
+version: 8.9.0
+entries:
+    - kind: security
+      summary: Use verified base image for building Fleet Server binaries
+      description: ""
+      component: ""
+      pr:
+        - "2339"
+      issue: []
+      timestamp: 1682696415
+      file:
+        name: 1682696415-crossbuild.yaml
+        checksum: 159350f6e8e2e60912b5ca57c2d0ee3fe10b1026
+    - kind: breaking-change
+      summary: API default error is now 500
+      description: Up until now when any not expected error was encountered it resulted int Bad Request response.\n This is changing and any not expected error will return Internal Server Error while keeping most of the current behavior\n unchanged and returning Bad Request on expected failure paths (e.g. Agent Inactive, Missing Agent ID, Missing Auth Header).
+      component: ""
+      pr:
+        - "2531"
+      issue:
+        - "2417"
+      timestamp: 1683024407
+      file:
+        name: 1683024407-API-default-error-is-now-500.yaml
+        checksum: 6db2475a1815378713dd450298d696612f02cf7c
+    - kind: bug-fix
+      summary: fix-action-seq-no
+      description: Fix for a bug where agent `action_seq_no` was overwritten with 0 if the `ackToken` was not provided.
+      component: ""
+      pr:
+        - "2582"
+      issue:
+        - "2519"
+      timestamp: 1684151814
+      file:
+        name: 1684151814-fix-action-seq-no.yaml
+        checksum: aac8471c8947618ba2e4c5fb1b21887118e923ef
+    - kind: enhancement
+      summary: Expose prometheus metrics
+      description: |
+        Expose prometheus metrics on metrics listener (when enabled).
+        Ship prometheus metrics with apm.Tracer when tracer is enabled.
```

Comment:
There's a space missing before "Tracer", but also the second sentence is not very clear to me, not sure if it needs to be rephrased.

Reply:
Actually there is no missing space, it is meant to be "Ship prometheus metrics with `apm.Tracer` when tracer is enabled."

Reply:
oh ok, it wasn't really clear, maybe adding single quotes would make it clearer? 

`Ship prometheus metrics with 'apm.Tracer' when tracer is enabled.`